bluebottle/assignments/serializers.py

```diff
@@ -27,13 +27,16 @@ class Meta(BaseActivityListSerializer.Meta):
         fields = BaseActivityListSerializer.Meta.fields + (
             'is_online',
             'end_date',
+            'date',
             'end_date_type',
             'registration_deadline',
             'capacity',
             'expertise',
             'duration',
             'location',
             'permissions',
+            'start_time',
```

Comment:
Through the admin you are only exposing `start` and here only `start_time`. What would happen if you change `start` in admin? You won't see those changes in the front-end, right?
Same for the other way around.

Reply:
Start time could just go. I kept it in so that during the release, reads do not break. The admin should just expose date